diff --git a/CMakeLists.txt b/CMakeLists.txt
index de38d5fe..5f2ec4b1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,10 +211,10 @@ macro(commom_lib_settings lib write )
         $<INSTALL_INTERFACE:include>

         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/basisu/transcoder>
-        $<INSTALL_INTERFACE:lib/basisu/transcoder>
+        #$<INSTALL_INTERFACE:lib/basisu/transcoder>

         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/other_include>
-        $<INSTALL_INTERFACE:other_include>
+        #$<INSTALL_INTERFACE:other_include>
     )

     if( LIB_TYPE STREQUAL STATIC )
@@ -353,7 +353,7 @@ macro(commom_lib_settings lib write )
             ${lib}
         PUBLIC
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/dfdutils>
-            $<INSTALL_INTERFACE:lib/dfdutils>
+            #$<INSTALL_INTERFACE:lib/dfdutils>
         )

         get_target_property( KTX_PUBLIC_HEADER ${lib} PUBLIC_HEADER )
@@ -428,10 +428,10 @@ target_include_directories(
     ktx
 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib>
-    $<INSTALL_INTERFACE:lib>
+    #$<INSTALL_INTERFACE:lib>

     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/basisu>
-    $<INSTALL_INTERFACE:lib/basisu>
+    #$<INSTALL_INTERFACE:lib/basisu>
 )
 target_compile_definitions(
     ktx
